q2_moshpit/busco/busco.py

```diff
@@ -0,0 +1,121 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2023, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+
+import os
+import tempfile
+import q2_moshpit.busco.utils
+from q2_moshpit.busco.utils import (
+    _parse_busco_params,
+    _render_html,
+)
+from q2_moshpit._utils import _process_common_input_params
+from typing import List
+from q2_types_genomics.per_sample_data._format import MultiMAGSequencesDirFmt
+
+
+def busco(
```

Comment:
I'm not sure I commented on this already but could you please rename this action to something like `evaluate-busco`? We usually try to follow the convention of using a verb (in imperative) in the beginning, followed by the tool name (where necessary/applicable).

Reply:
How about `run-busco`? Since BUSCO stands for Benchmarking Universal Single-Copy Ortholog assessment tool it would be something like "run-assessment-tool" instead of "evaluate-assessment-tool"

Reply:
Hmmmm, actually... BUSCO itself doesn't have the "assessment tool" as part of the abbreviation. I suggested `evaluate` because it actually indicates to the user that it's a QC action and `busco` indicates which tool is used for evaluation. If you look at the other actions in this plugin (and others), you'll notice this convention followed by many other actions.

Reply:
Ok makes sense :)